data/conditions.ts

```diff
@@ -708,8 +708,7 @@ export const Conditions: {[k: string]: ConditionData} = {
 			if (['cramorantgulping', 'cramorantgorging'].includes(pokemon.species.id) && !pokemon.transformed) {
 				pokemon.formeChange('cramorant');
 			}
-			this.add('-start', pokemon, 'Dynamax');
-			if (pokemon.gigantamax) this.add('-formechange', pokemon, pokemon.species.name + '-Gmax');
+			this.add('-start', pokemon, 'Dynamax', pokemon.gigantamax ? 'Gmax' : '');
```

Comment:
Even though this got approved already, would it be a good idea to only send "G" rather than "Gmax" (like how packed teams work)? Since the client will only care about the argument not being an empty string.

Reply:
I don't think this is necessary. The protocol doesn't pack quite this tightly anywhere else.